ibm/mas_devops/roles/cp4d_wds/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: "Fail if mas_instance_id is not provided"
+  when: mas_instance_id is not defined or mas_instance_id == ""
+  fail:
+    msg: "mas_instance_id property is required"
+
+- name: "Fail if mas_config_dir is not provided"
+  when: mas_config_dir is not defined or mas_config_dir == ""
+  fail:
+    msg: "mas_config_dir property is required"
+
+- name: "Fail if  is not provided"
+  when:
+    - assist_wds_pass is not defined or assist_wds_pass == ""
+    - useexternalwds is defined and useexternalwds
+  fail:
+    msg: "assist_wds_pass property is required"
+
+- name: "Fail if  is not provided"
+  when:
+    - assist_wds_admin is not defined or assist_wds_admin == ""
+    - useexternalwds is defined and useexternalwds
+  fail:
+    msg: "assist_wds_admin property is required"
+
+- name: "Fail if  is not provided"
+  when:
+    - assist_wds_url is not defined or assist_wds_url == ""
+    - useexternalwds is defined and useexternalwds
+  fail:
+    msg: "assist_wds_url property is required"
+
+
+- name: "cp4d/wds: Query CP4D Operator namespace"
```

Comment:
This name doesn't seem to match what the task is actually doing

Reply:
update with more accurate description